helix-core/src/main/java/org/apache/helix/HelixAdmin.java

```diff
@@ -228,6 +228,16 @@ void addResource(String clusterName, String resourceName, int numPartitions, Str
    */
   void dropInstance(String clusterName, InstanceConfig instanceConfig);
 
+  /**
+   * Purge offline instances that have been offline for longer than the offline duration time
+   * from a cluster
+   * @param clusterName
+   * @param offlineTimeout the threshold of offline time to decide whether an offline instance
```

Comment:
"the threshold of offline time to"... -> this is a little confusing. the fact that we are using timeout to describe this could be confusing to the reader. 

timeout is a term used when one actively counts down to see if an operation completes or not. in this case, as you clarified, we are not actively counting down or waiting on anything. so the use of timeout is still misleading. my suggestion is to use offlineDuration as that wouldn't cause any confusion or give readers a wrong idea.

for example, something like this would work:

`@param offlineDuration if an offline instance has been offline for longer than the set offlineDuration, the offline instance becomes eligible for being purged/deleted`